chain/client/src/tests/catching_up.rs

```diff
@@ -694,7 +694,7 @@ enum ChunkGrievingPhases {
 
 // TODO(#3180): seals are disabled in single shard setting
 #[test]
-#[cfg_attr(not(feature = "expensive_tests"), ignore)]
+// #[cfg_attr(not(feature = "expensive_tests"), ignore)]
```

Comment:
Why is this commented out? It's a but puzzling to see newly commented out code in a review.

Reply:
@matklad  With `1.58.0.`, this generates a warning about unused `ignore`.

Reply:
Ah I see, we have a blanket `#[ignore]` at the next line. Let's just remove the commented-out line -- I see how it communicates "when you unignore the test, make sure to mark it as expensive", but I don't think that adds enough real value -- people would just be puzzled over this.